Cargo.toml

```diff
@@ -316,6 +316,9 @@ pbr_multi_layer_material_textures = [
 # Enable support for anisotropy texture in the `StandardMaterial`, at the risk of blowing past the global, per-shader texture limit on older/lower-end GPUs
 pbr_anisotropy_texture = ["bevy_internal/pbr_anisotropy_texture"]
 
+# Enable support for spectral colors (monochromatic light) in lights
+spectral_lighting = ["bevy_internal/spectral_lighting"]
```

Comment:
Why make this a cargo feature? Imo, the less features we have, the better, and we should only use them for things that can't be changed at runtime.

Can we make this a field on PbrPlugin instead?